crux_core/src/capabilities/compose.rs

```diff
@@ -157,4 +157,14 @@ impl<Ev> Capability<Ev> for Compose<Ev> {
     {
         Compose::new(self.context.map_event(f))
     }
+
+    #[cfg(feature = "typegen")]
+    fn register_types(_generator: &mut crate::typegen::TypeGen) -> crate::typegen::Result {
+        panic!(
+            r#"
+            The Compose Capability should not be registered for type generation.
+            Instead, use #[effect(skip)] to skip the generation of an effect variant for the Compose Capability.
+            "#
```

Comment:
Any consequence to just silently doing nothing here instead of panicing?

Reply:
The typegen blows up on the `Never` operation type, so figured panicking is an opportunity to prompt using the effect skip attirbute